src/bootstrap.php

```diff
@@ -202,7 +202,9 @@
 
     $recommendations = $relations;
 
-    $appendFirstThatDoesNotAlreadyExist = function (Sequence $recommendations, Sequence $toInsert) : Sequence {
+    $appendFirstThatDoesNotAlreadyExist = function (Sequence $recommendations, Sequence $toInsert, int $max = 1) : Sequence {
```

Comment:
the name of `max` correctly reflects that is `$toInsert` is too small, less than `$max` items may be added. Which is unlikely given the size of subjects anyway.